pyne/utils.py

```diff
@@ -293,6 +294,32 @@ def line_almost_same(l1, l2, rel_tol=1e-9):
         return True
 
 
+def str_elements_in_line(s, line):
```

Comment:
I will try "regex" later in new updates.

Reply:
I remove the function `str_elements_in_line()` (in https://github.com/pyne/pyne/pull/1123/commits/c11ab7711f1a0e019738284386e53e76c7f2b066) because this function is only used for matching strings when read ALARA output. It is fragile and may cause errors if someone uses it to other places.

For matching strings of specific lines for ALARA, a simple `some_string in line` is enough.